### DISCUSSION TITLE
I am unable to import the products using .csv file from medusa-admin product page

### DISCUSSION THREAD

**Discussion by HarkiratShopism1** [medusajs/medusa#4583] [Q&A]
I have installed the minio file system I am able to export the products but I am not able to import the products to the backend by uploading a csv file in the medusa admin product page. When I try to upload the csv file to the medusa admin I get notified with an error that batch job cancelled. Can someone please look into it and  help me out resolve this issue.

Thanks

**Comment by fPolic** [comment]
hey @HarkiratShopism1,
Could you please open an issue [here](https://github.com/medusajs/medusa/issues/new/choose) with a description of your setup and share the error message you are getting?
